python-packages/order_utils/setup.py

```diff
@@ -0,0 +1,119 @@
+"""setuptools module for order_utils package."""
+
+import subprocess  # nosec
```

Comment:
What does `nosec` mean?

Reply:
it's an ignore directive for the `bandit` linter, which is focused on security issues, so "nosec" means "no security checks".